src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -1890,7 +1891,7 @@ $$C_{req} = N_m \times D_j \times \frac{1}{P_m}$$
 When using these formulas, remember to account for these factors:
 
   * **Job duration ($D_j$):** Your average should include jobs that **time out** (often \~3 minutes), as these hold a core for their entire duration.
```

Comment:
* **Job duration ($D_{avg,m}$):** Your average should include jobs that **time out** (often \~3 minutes), as these hold a core for their entire duration.